### PR TITLE
Fix config

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | -

### CODE REVIEW

**Review by vjik — COMMENTED** [review]
May be add config tests to prevent such problems in the future?

**Comment by rustamwin** [comment]
> May be add config tests to prevent such problems in the future?

For all packages?

**Comment by vjik** [comment]
> > May be add config tests to prevent such problems in the future?
> 
> For all packages?

Some packages already have config tests. Suggest make config test to `yiisoft/log-target-file` in this PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## 2.0.1 under development
 
-- no changes in this release.
+- Fix #45: Fix definition config (@rustamwin)
```
Comment (vjik) [inline CHANGELOG.md:5]:
```suggestion
- Fix #45: Fix definitions config (@rustamwin)
```

**Review by vjik — APPROVED** [review]

**Comment by samdark** [comment]
https://github.com/yiisoft/log-target-file/runs/7488812096?check_suite_focus=true#step:9:21

**Pull request merged**